ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -662,7 +649,7 @@ SMS_PREFIXES_BY_COUNTRY = '{{ localsettings.SMS_PREFIXES_BY_COUNTRY }}'
 {% endif %}
 
 {% if localsettings.get('MIA_THE_DEPLOY_BOT') %}
-MIA_THE_DEPLOY_BOT = '{{ localsettings.MIA_THE_DEPLOY_BOT }}'
+MIA_THE_DEPLOY_BOT_API = '{{ localsettings.MIA_THE_DEPLOY_BOT }}'
```

Comment:
actually that depends if you're planning on changing the name in the `config` dir